reconcile/test/test_terraform_resources.py

```diff
@@ -1,6 +1,27 @@
+import pytest
+
 import reconcile.terraform_resources as integ
 
 
+def test_cannot_pass_two_aws_account_if_not_dry_run():
+    with pytest.raises(RuntimeError) as excinfo:
+        integ.run(False, account_name=("a", "b"))
+
+    assert "Running with multiple accounts is only supported in dry-run mode" in str(
+        excinfo.value
+    )
+
+
+def test_cannot_pass_invalid_aws_account(mocker):
+    mocker.patch("reconcile.queries.get_aws_accounts", return_value=[{"name": "a"}])
```

Comment:
`autospec=True` in patch will ensure that you pass the correct number of arguments. It's generally a best practice for all patching.